RoboJackets/igvc-software
# Feat/scan to pointcloud node cpp

# Description

Separates the scan_to_pointcloud node into a header file and a C++ class.


# Testing steps (If relevant)
1. Run simulation on noetic-migration branch
2. Run simulation on feat/scan_to_pointcloud_node_cpp branch
3. Check that the results for the scan_to_pointcloud node are the same for both branches.

Expectation: The scan_to_pointcloud node should maintain functionality

# Self Checklist
